lib/models/tree.model.ts

```diff
@@ -313,31 +325,43 @@ export class TreeModel implements ITreeModel, OnDestroy {
   }
 
   @action setExpandedNode(node, value) {
-    this.expandedNodeIds = Object.assign({}, this.expandedNodeIds, {[node.id]: value});
-    this.fireEvent({ eventName: TREE_EVENTS.toggleExpanded, node, isExpanded: value });
+    this.expandedNodeIds = Object.assign({}, this.expandedNodeIds, {
+      [node.id]: value
+    });
+    this.fireEvent({
+      eventName: TREE_EVENTS.toggleExpanded,
+      node,
+      isExpanded: value
+    });
   }
 
   @action expandAll() {
-    this.roots.forEach((root) => root.expandAll());
+    this.roots.forEach(root => root.expandAll());
   }
 
   @action collapseAll() {
-    this.roots.forEach((root) => root.collapseAll());
+    this.roots.forEach(root => root.collapseAll());
   }
 
   @action setIsHidden(node, value) {
-    this.hiddenNodeIds = Object.assign({}, this.hiddenNodeIds, {[node.id]: value});
+    this.hiddenNodeIds = Object.assign({}, this.hiddenNodeIds, {
+      [node.id]: value
+    });
   }
 
   @action setHiddenNodeIds(nodeIds) {
-    this.hiddenNodeIds = nodeIds.reduce((hiddenNodeIds, id) => Object.assign(hiddenNodeIds, {
-      [id]: true
-    }), {});
+    this.hiddenNodeIds = nodeIds.reduce(
+      (hiddenNodeIds, id) =>
+        Object.assign(hiddenNodeIds, {
+          [id]: true
+        }),
+      {}
+    );
   }
 
   performKeyAction(node, $event) {
-    const action = this.options.actionMapping.keys[$event.keyCode];
-    if (action) {
+    const performKeyAction = this.options.actionMapping.keys[$event.keyCode];
+    if (performKeyAction) {
       $event.preventDefault();
       action(this, node, $event);
```

Comment:
You renamed the variable but didn't change the references to it.
For example in line 366.
You should rename variables using the IDE (like in VSCode you have "rename symbol" option)

Reply:
Fixed; Didn't initially understand the how action at line 366 worked in conjunction with the const action but digging a bit deeper I get it now.